e2e/cypress/e2e/navigationdropdown.cy.js

```diff
@@ -5,17 +5,36 @@ describe("Dropdown navigation on small screens", () => {
   });
 
   it("should show the profile menu", () => {
-    cy.get('[data-cy="toggle-text"]').contains("Profilmenu").click();
+    cy.get('[data-cy="menu-title"]').contains("Profilmenu").click();
     cy.get('[data-cy="mobile-link-myLibraries"]').click();
-    cy.get('[data-cy="mobile-link-myLibraries"]')
-      .parent()
-      .should("have.css", "background-color", "rgb(51, 51, 255)");
+    cy.get('[data-cy="mobile-link-myLibraries"]').should(
+      "have.css",
+      "background-color",
+      "rgb(242, 242, 242)"
+    );
```

Comment:
Hvad tester den her? For hvis det er, at den er focus, så kan man jo bruge `cy.focused()` i stedet.

Reply:
den tester at selectede links har en bestemt baggrundsfarve, så det er ikke relateret til focus.